packages/date/src/DateRange.js

```diff
@@ -265,7 +319,7 @@ const DateRange = ({
           disabled={attributes.disabled}
           onFocusChange={onFocusChange}
           renderCalendarInfo={renderDateRanges}
-          customArrowIcon="-"
+          customArrowIcon={customArrowIcon}
```

Comment:
should this be added to default props to remain backwards compatible?

Reply:
I don't think it has to be because when `customArrowIcon` is `undefined` then react-dates provides the default svg arrow icon for us.